services/PA-accounts-service/src/user/user.service.ts

```diff
@@ -73,6 +74,13 @@ export class UserService {
     }
   }
 
+  public async setDid(id: number, didDto: DidDto): Promise<UserRO> {
+    let user = await this.userRepository.findOne(id);
+    user.did = didDto.did;
+    const updatedUser = await this.userRepository.save(user);
+    return this.buildUserRO(updatedUser);
```

Comment:
Does it need to return the full User-object?
It is only a "POST"-endpoint right, we only need confirmation that it succeeded, nothing more.